packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -389,18 +423,22 @@ export class AuroraPostgresEngineVersion {
    * @param auroraPostgresMajorVersion the major version of the engine,
    *   for example "9.6"
    */
-  public static of(auroraPostgresFullVersion: string, auroraPostgresMajorVersion: string): AuroraPostgresEngineVersion {
-    return new AuroraPostgresEngineVersion(auroraPostgresFullVersion, auroraPostgresMajorVersion);
+  public static of(auroraPostgresFullVersion: string, auroraPostgresMajorVersion: string,
+    auroraPostgresFeatures?: EngineFeatures): AuroraPostgresEngineVersion {
```

Comment:
Can you leave an empty line when the signature and body are indented on the same level? Makes it more readable.